File: Python/sample_sender.py
import imp
import multiprocessing
import time
import communicator
import csv
import os.path
import platform


communicator = communicator.Communicator("youcanchangethis")
communicator.setdefaultreceiver("127.0.0.1",1234)

message="Hello! ;)"

OS={
  
  "name" : platform.system(),
  "version" : platform.release()
}

Lang={
  "name" : "Python",
  "version" : platform.python_version()
}

sendingImage=buffer(open("./logo image/python logo.png", "rb").read())

communicator.send({"msg":message,"os":OS["name"],"osversion":OS["version"],"lang":Lang["name"],"langversion":Lang["version"],"senderimg":sendingImage  },"TestMessage")
  
print ("Message was sent")

raw_input()

File: Node.js/sample_sender.js
var os=require("os");
var fs=require("fs");

var Communicator = require("./Library/Communicator/index.js");

var communicator = new Communicator();

communicator.config({ password: "youcanchangethis"});

communicator.setdefaultreceiver("127.0.0.1", 1234);

var osnames={"win32":"Windows","drawin":"MacOS","linux":"Linux"};

var OS=
{
  platform:osnames[os.platform()],
  version:os.release()
};

var Lang=
{
  name:"Node.js",
  version:process.versions.node
}

var message="Hello! ;)";

var sendingImage=fs.readFileSync("./logo image/nodejs logo.png");

communicator.send({msg:message,os:OS.platform,osversion:OS.version,lang:Lang.name,langversion:Lang.version,senderimg:sendingImage}, { type: "TestMessage"}).then(() =>
{
  console.log("Message was sent"); 
})